newrelic-agent/src/main/java/com/newrelic/agent/jmx/LinkingMetadata.java

```diff
@@ -0,0 +1,19 @@
+package com.newrelic.agent.jmx;
+
+import com.newrelic.api.agent.Agent;
+import com.newrelic.api.agent.Logger;
+import com.newrelic.api.agent.NewRelic;
+
+import java.util.Map;
+import java.util.logging.Level;
+
+public class LinkingMetadata implements LinkingMetadataMBean {
+
+    @Override
+    public Map<String, String> readLinkingMetadata() {
+        Agent agent = NewRelic.getAgent();
+        Logger logger = agent.getLogger();
+        logger.log(Level.INFO, "JmxLinkingMetadata: Fetching linking metadata from the agent...");
```

Comment:
```suggestion
        logger.log(Level.INFO, "JMX LinkingMetadata: Fetching linking metadata from the agent...");
```

I guess we should be consistent with our logging and use `JMX LinkingMetadata`